### PR TITLE
Messagelings

### PR DESCRIPTION
**Pull request by Neloop**

### CODE REVIEW

File: app/config/permissions.neon [hunk 1]
```diff
@@ -767,3 +767,48 @@ permissions:
...
+    ############################
+    # Notification permissions #
+    ############################
+
+    - allow: true
+      role: scope-read-all
+      resource: notification
+      actions:
+        - viewAll
+        - viewCurrent
```
[the remark this answers is not in the document]
Comment (Neloop) [reply app/config/permissions.neon]: You are absolutely and positively right

File: app/V1Module/security/Roles.php [hunk 2]
```diff
@@ -2,40 +2,50 @@
...
 
   /**
    * Validate given role against available user roles.
    * @param string $role
    * @return bool true if given role is valid
    */
-  public static function validateRole(string $role): bool {
-    if (in_array($role, self::ROLES)) {
+  public function validateRole(string $role): bool {
+    if (in_array($role, array_keys($this->roles))) {
```
[the remark this answers is not in the document]
Comment (Neloop) [reply app/V1Module/security/Roles.php]: yup... I left almost all original code with minor change and did not spot this simplification

File: app/model/entity/LocalizedGroup.php [hunk 3]
```diff
@@ -31,10 +31,10 @@ class LocalizedGroup extends LocalizedEntity implements JsonSerializable {
    */
   protected $group;
 
-  public function __construct($locale, string $name, string $description, ?LocalizedGroup $createdFrom = null) {
+  public function __construct($locale, string $name, string $text, ?LocalizedGroup $createdFrom = null) {
     parent::__construct($locale);
     $this->name = $name;
-    $this->description = $description;
+    $this->description = $text;
```
Comment (krulis-martin) [inline app/model/entity/LocalizedGroup.php]: Is there some reason to rename argument `text`, when it is being assigned to `description`?

Comment (Neloop) [reply app/model/entity/LocalizedGroup.php]: Interesting, that was prolly IDE

**Comment by Neloop** [comment]
Ready to re-review

**Pull request merged**